--- Feathercoin-0.9.6/src/qt/qimagesource.cpp.orig	2017-01-30 17:11:35.000000000 +0100
+++ Feathercoin-0.9.6/src/qt/qimagesource.cpp	2017-04-16 22:11:31.628803262 +0200
@@ -16,11 +16,11 @@
 
 ArrayRef<char> QImageLuminanceSource::getMatrix() const
 {
-    ArrayRef<char> mymatrix(width*height);
-    for (int y = 0; y < height; y++)
+    ArrayRef<char> mymatrix(getWidth()*getHeight());
+    for (int y = 0; y < getHeight(); y++)
     {
-        for (int x = 0; x < width; x++) {
-            mymatrix[y*width+x] = qGray(image.pixel(x, y));
+        for (int x = 0; x < getWidth(); x++) {
+            mymatrix[y*getWidth()+x] = qGray(image.pixel(x, y));
         }
     }
     return mymatrix;
