--- Feathercoin-0.19.1/configure.ac.orig	2020-08-06 11:29:39.000000000 +0200
+++ Feathercoin-0.19.1/configure.ac	2020-09-21 22:22:56.190904468 +0200
@@ -1252,6 +1252,7 @@
       else
           AC_DEFINE_UNQUOTED([ENABLE_ZMQ],[0],[Define to 1 to enable ZMQ functions])
       fi
+      PKG_CHECK_MODULES([SECP256K1], [libsecp256k1],, [AC_MSG_ERROR(libsecp256k1 not found.)])
     ]
   )
 else
@@ -1655,9 +1656,6 @@
   AC_CONFIG_SUBDIRS([src/univalue])
 fi
 
-ac_configure_args="${ac_configure_args} --disable-shared --with-pic --enable-benchmark=no --with-bignum=no --enable-module-recovery --disable-jni"
-AC_CONFIG_SUBDIRS([src/secp256k1])
-
 AC_OUTPUT
 
 dnl Taken from https://wiki.debian.org/RpathIssue
--- Feathercoin-0.16.3/src/Makefile.am~	2018-09-22 09:16:24.000000000 +0200
+++ Feathercoin-0.16.3/src/Makefile.am	2018-09-25 20:40:02.844966767 +0200
@@ -2,7 +2,7 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-DIST_SUBDIRS = secp256k1 univalue
+DIST_SUBDIRS = univalue
 
 AM_LDFLAGS = $(PTHREAD_CFLAGS) $(LIBTOOL_LDFLAGS) $(HARDENED_LDFLAGS)
 AM_CXXFLAGS = $(HARDENED_CXXFLAGS) $(ERROR_CXXFLAGS)
@@ -20,7 +20,7 @@ endif
 
 BITCOIN_INCLUDES=-I$(builddir) $(BDB_CPPFLAGS) $(BOOST_CPPFLAGS) $(LEVELDB_CPPFLAGS) $(CRYPTO_CFLAGS) $(SSL_CFLAGS)
 
-BITCOIN_INCLUDES += -I$(srcdir)/secp256k1/include
+BITCOIN_INCLUDES += $(SECP256K1_CFLAGS)
 BITCOIN_INCLUDES += $(UNIVALUE_CFLAGS)
 
 LIBBITCOIN_SERVER=libbitcoin_server.a
@@ -30,8 +30,8 @@ LIBBITCOIN_CLI=libbitcoin_cli.a
 LIBBITCOIN_UTIL=libbitcoin_util.a
 LIBBITCOIN_CRYPTO=crypto/libbitcoin_crypto.a
 LIBBITCOINQT=qt/libbitcoinqt.a
-LIBSECP256K1=secp256k1/libsecp256k1.la
-
+LIBSECP256K1=$(SECP256K1_LIBS)
+
 if ENABLE_ZMQ
 LIBBITCOIN_ZMQ=libbitcoin_zmq.a
 endif
